tests/__init__.py

```diff
@@ -1,132 +1,19 @@
-        self.raw_input_cache = __builtin__.raw_input
-        __builtin__.raw_input = self.get_next_value
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        __builtin__.raw_input = self.raw_input_cache
-
-
-class CloudBaseTest(unittest.TestCase):
-
-    def setUp(self):
-        self.resource_path = RESOURCE_PATH
-
-        # A hack to prevent get_application_default from going GAE route.
-        self._server_software_org = os.environ.get('SERVER_SOFTWARE')
-        os.environ['SERVER_SOFTWARE'] = ''
-
-        # Constants from environment
-        test_bucket_name = os.environ.get(BUCKET_NAME_ENV, '')
-        test_project_id = os.environ.get(PROJECT_ID_ENV, '')
-        if not test_project_id or not test_bucket_name:
-            raise Exception('You need to define an env var "%s" and "%s" to '
-                            'run the test.'
-                            % (PROJECT_ID_ENV, BUCKET_NAME_ENV))
-
-        # Constants from resources/constants.json
-        with open(
-                os.path.join(RESOURCE_PATH, 'constants.json'),
-                'r') as constants_file:
-
-            self.constants = json.load(constants_file)
-        self.constants['projectId'] = test_project_id
-        self.constants['bucketName'] = test_bucket_name
-        self.constants['cloudStorageInputURI'] = (
-            self.constants['cloudStorageInputURI'] % test_bucket_name)
-        self.constants['cloudStorageOutputURI'] = (
-            self.constants['cloudStorageOutputURI'] % test_bucket_name)
-
-    def tearDown(self):
-        if self._server_software_org:
-            os.environ['SERVER_SOFTWARE'] = self._server_software_org
-
-
-class DatastoreTestbedCase(unittest.TestCase):
-    """A base test case for common setup/teardown tasks for test."""
-    def setUp(self):
-        if not APPENGINE_AVAILABLE:
-            raise SkipTest()
-
-        """Setup the datastore and memcache stub."""
-        # First, create an instance of the Testbed class.
-        self.testbed = testbed.Testbed()
-        # Then activate the testbed, which prepares the service stubs for
-        # use.
-        self.testbed.activate()
-        # Create a consistency policy that will simulate the High
-        # Replication consistency model.
-        self.policy = datastore_stub_util.PseudoRandomHRConsistencyPolicy(
-            probability=0)
-        # Initialize the datastore stub with this policy.
-        self.testbed.init_datastore_v3_stub(
-            datastore_file=tempfile.mkstemp()[1],
-            consistency_policy=self.policy)
-        self.testbed.init_memcache_stub()
-
-    def tearDown(self):
-        self.testbed.deactivate()
-
-
-@contextlib.contextmanager
-def capture_stdout():
-    """Capture stdout."""
-    fake_stdout = StringIO.StringIO()
-    old_stdout = sys.stdout
-
-    try:
-        sys.stdout = fake_stdout
-        yield fake_stdout
-    finally:
-        sys.stdout = old_stdout
+from .utils import (
+    BUCKET_NAME_ENV,
```

Comment:
I moved all of the code in ``__init__.py`` to ``utils.py``, and added import aliases to the items in ``uitls.py``. This is standard python stuff to find in ``__init__.py``, and doesn't warrant further explanation.